scout/server/blueprints/institutes/templates/overview/phenomodels.html

```diff
@@ -122,10 +163,15 @@ <h5 class="card-title">Create new phenotype model</h5>
 
 {% block scripts %}
 {{ super() }}
+<script src="https://cdn.jsdelivr.net/npm/bootstrap-select@1.13.9/dist/js/bootstrap-select.min.js"></script>
 <script>
   $(document).on("click", ".removeModel", function () {
     var modelId = $(this).data('id');
     $('input#model_id').val(modelId);
 });
+$('select[multiple]').selectpicker({
```

Comment:
Not in like css? Ok either way I guess.

Reply:
Bah it's quite easy to have it this way..

Reply:
But there is actually an error in the code, too many }); 
Good that you pointed it out!